### PR TITLE
Support PDF Surface

### PR DESCRIPTION
**Pull request by yagince**
hi @mvz

i implemented cairo pdf surface

### CODE REVIEW

File: lib/gir_ffi-cairo/pdf_surface.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Cairo
```
Comment (mvz) [inline lib/gir_ffi-cairo/pdf_surface.rb]: You should add `load_class :PDFSurface` here.

File: lib/gir_ffi-cairo/pdf_surface.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  class PDFSurface < Surface
```
Comment (mvz) [inline lib/gir_ffi-cairo/pdf_surface.rb]: Having added `load_class :PDFSurface` above, you can remove `< Surface` here.

**Comment by mvz** [comment]
Great! If you can fix the one bit I commented on, this can be merged, thanks.

File: lib/gir_ffi-cairo/pdf_surface.rb (hunk 1 above)
Comment (yagince) [inline lib/gir_ffi-cairo/pdf_surface.rb]: Thank you for your comment.

I tried to fix as follows, but an error has occurred.
```ruby
module Cairo
  load_class :PDFSurface

  class PDFSurface
    def self.create filename, width, height
      ptr = Lib.cairo_pdf_surface_create filename, width, height
      surface = self.wrap ptr

      if block_given?
        yield surface
        surface.finish
      end

      surface
    end

    def set_size(width, height)
      Lib.cairo_pdf_surface_set_size(self, width, height)
    end
  end

  module Lib
    attach_function :cairo_pdf_surface_create, [:string, :double, :double], :pointer
    attach_function :cairo_pdf_surface_set_size, [:pointer, :double, :double], :void
  end
end
```

```
/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/gir_ffi-0.10.0/lib/gir_ffi/builders/module_builder.rb:45:in `find_namespaced_class_info': Class PDFSurface not found in namespace cairo (NameError)
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/gir_ffi-0.10.0/lib/gir_ffi/builders/module_builder.rb:36:in `build_namespaced_class'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/gir_ffi-0.10.0/lib/gir_ffi/module_base.rb:16:in `load_class'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/lib/gir_ffi-cairo/pdf_surface.rb:2:in `<module:Cairo>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/lib/gir_ffi-cairo/pdf_surface.rb:1:in `<top (required)>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/lib/gir_ffi-cairo.rb:9:in `require'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/lib/gir_ffi-cairo.rb:9:in `<top (required)>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/test/test_helper.rb:3:in `require'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/test/test_helper.rb:3:in `<top (required)>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/test/gir_ffi-cairo/context_test.rb:1:in `require'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/test/gir_ffi-cairo/context_test.rb:1:in `<top (required)>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:15:in `require'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:15:in `block in <main>'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:4:in `select'
	from /Users/natsuki/workspace/opensources/gir_ffi-cairo/vendor/bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:4:in `<main>'
```

`Cairo::ImageSurface` is inherits the `Surface` without a `load_class`,
PDFSurface is will be better to use the `load_class`?

Comment (mvz) [inline lib/gir_ffi-cairo/pdf_surface.rb]: Oh, you're right. PDFSurface isn't defined as a type by Cairo.

**Comment by mvz** [comment]
Merged, thanks!

**Comment by yagince** [comment]
thanks :)

**Pull request merged**